packages/mdl-card/README.md

```diff
@@ -0,0 +1,110 @@
+# mdl-card
+
+MDL card is a component that implements the
+[Material Design card component](https://spec.googleplex.com/quantum/components/cards.html), and makes it available to
+developers as a set of CSS classes.
+
+## Installation
+
+> Note: Installation via the npm registry will be available after alpha.
+
+
+## Usage
+
+```html
+<div class="mdl-card demo-card">
+  <div class="mdl-card__primary">
+    <div class="mdl-card--large-title">Title goes here</div>
+    <div class="mdl-card--subtitle">Subtitle here</div>
+  </div>
+  <div class="mdl-card__supporting-text">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+    veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+    commodo consequat.
+  </div>
+  <div class="mdl-card__actions">
+    <button class="mdl-button">Action 1</button>
+    <button class="mdl-button">Action 2</button>
+  </div>
+</div>
+```
+
+### Content blocks
+
+Cards are composed of different content blocks, which are typically laid out in vertical succession.
+
+#### Rich media
+
+```css
+#example .mdl-card__media {
+  background-image: url("pretty.jpg");
+  background-size: 100%;
+  height: 90px;
+}
+```
+
+```html
+<div class="mdl-card__media"></div>
+```
+
+This area is used for showing rich media in cards. Use the `mdl-card__media` CSS class.
+
+
+#### Actions
+
+```html
+<div class="mdl-card__actions">
+  <button class="mdl-button">Action 1</button>
+  <button class="mdl-button">Action 2</button>
+</div>
+```
+
+This area is used for showing different actions the user can take. It's typically used with buttons, as in the example
+above, or with icon buttons, as below:
+
+> TODO(sgomes): Add icon button support and example.
+
+You can use the `mdl-card__actions--vertical` option to lay actions out vertically instead of horizontally:
+
+```html
+<div class="mdl-card__actions mdl-card__actions--vertical">
+  <button class="mdl-button">Action 1</button>
+  <button class="mdl-button">Action 2</button>
+</div>
+```
+
+#### Primary title / text
+
+```html
+<div class="mdl-card__primary">
```

Comment:
ah yeah so I'd stick with `primary` then, or maybe `primary-section`. I agree that having `-text` would be confusing. As a side note, we may want to consider using `<section>` tags here instead of divs to make the markup more semantic.